src/utils/sourcemaps/inject.rs

```diff
@@ -106,6 +115,36 @@ impl fmt::Display for InjectReport {
     }
 }
 
+pub fn inject_debug_id(js_source: &str, debug_id: DebugId) -> String {
```

Comment:
I think this function should have a more indicative name.

Reply:
What do you think about `inject_into_compiled_source`? If you don't like that name, could you suggest something?

Reply:
ah okay got it – tbh I prefer `inject…` naming, since I found the `fixup` naming a bit confusing (what exactly are we "fixing up"?) but for consistency I suppose I can change the naming<br><br>long term, I'd anyways like us to just use this new function for all injection occurring at the beginning of the file

Reply:
@loewenheim I decided to explicitly deprecate the existing `fixup_js_file` (without renaming, in this PR), and I'll rename `fixup_js_file_end` to `inject_at_end` in #2475. This function I'll rename to `inject_at_start`. 

Is that okay with you?

Reply:
Yeah, that sounds good